src/list.js

```diff
@@ -0,0 +1,17 @@
+import Task from './task';
+
+export default class List {
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('tasks')) || List.initialize();
+  }
+
+  static initialize() {
+    const task1 = new Task('Eat', 0);
+    const task2 = new Task('Rest', 1);
+    const task3 = new Task('Sleep', 2);
+    return [task1, task2, task3];
+  }
```

Comment:
- [ ]  As per project requirements each task object should contain three keys. I can see there is one key missing like for `completed` that should be a boolean datatype. Kindly try to fix this.

Reply:
Have you seen the constructor?

>   constructor(desc, index) {
    this.desc = desc;
    this.completed = false;
    this.index = index;
    this.id = Task.generateID();
  }

Reply:
- Yes, I just noticed it. Everything looks perfect for now. you are good to go. Just raise a CR request.
Happy Coding.